template-parts/header_content-archive.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * Header content template for the 'Section' page template
+ */
+?>
+
+<?php
+global $post;
+
+$title_string = wptexturize( ucfwp_get_header_title( null ) );
```

Comment:
`ucfwp_get_header_title()` returns a texturized title--no need to call `wptexturize()` here

Reply:
Removed.